### PR TITLE
registerlogin front-end&back-end

### PR DESCRIPTION
**Pull request by kokioba95**
## Description / Purpose (checkpoints)
It is confirmation of this PR, Register function & layout, login function & layout.

Fixes issue: <https://docs.google.com/spreadsheets/d/1Ear__g58irVlaI_tpzWUkf__JRcBz8u2d2DdRYKMiDk/edit?usp=sharing> <Back Log #8>

## How To Test This PR
1. Go Register page (Ex.http://localhost:8086/register)
2. Input User register information
3. Click Register
4. (3)After, my_profile page
5. Press log out in the header right tab.
6. Go login page(Ex.http://localhost:8086/login)
7. Input User Credentials
8. Click Login 
 
 **Commands**
php artisan migrate
php artisan make:controller UserController

 **Screenshots**
 - <https://drive.google.com/drive/folders/1UX_at7VJu0OeH5Mb7Eo7ULK3hSw7Ws-W?usp=sharing>

### CODE REVIEW

File: resources/views/user_home.blade.php [hunk 1]
```diff
@@ -0,0 +1,6 @@
+@extends('layouts.app')
```
Comment (AubreyV) [inline resources/views/user_home.blade.php]: What does this blade file do?

File: routes/web.php [hunk 2]
```diff
@@ -16,5 +16,6 @@
 });
 Auth::routes();
 Route::get('/my_profile', 'HomeController@index')->name('my_profile');
+Route::get('/users', 'UserController@users');
```
Comment (AubreyV) [inline routes/web.php]: What is this for?

Comment (kokioba95) [reply routes/web.php]: I removed Code and
I removed UserController, which has no functional problem.

File: resources/views/user_home.blade.php (hunk 1 above)
Comment (kokioba95) [reply resources/views/user_home.blade.php]: I deleted because I was booking with other page.

File: routes/web.php (hunk 2 above)
Comment (AubreyV) [reply routes/web.php]: it's still here.

**Pull request merged**